spec/defines/config/aclpolicyfile_spec.rb

```diff
@@ -24,42 +23,45 @@
       },
       'for' => {
         'resource' => [
-          { 'equals' => {'kind' => 'project'}, 'allow' => ['create'] }
+          { 'equals' => { 'kind' => 'project' }, 'allow' => ['create'] }
         ]
       },
       'by' => [
-        {'groups' => ['admin']},
+        { 'groups' => ['admin'] },
       ]
     }
   ]
 
   context 'default parameters' do
     let(:title) { 'defaultPolicy' }
-    let(:params) {{
-      :acl_policies => test_policies
-    }}
+    let(:params) do
+      {
+        :acl_policies => test_policies
+      }
+    end
 
-    it { should contain_file('/etc/rundeck/defaultPolicy.aclpolicy').with({
-      'owner' => 'rundeck',
-      'group' => 'rundeck',
-      'mode'  => '0640',
-    })}
+    it do
+      should contain_file('/etc/rundeck/defaultPolicy.aclpolicy').with('owner' => 'rundeck',
+                                                                       'group' => 'rundeck',
+                                                                       'mode'  => '0640',)
```

Comment:
@jyaworski so  you want to just put the .with on a newline, and indent it twice. Its easier for us humans to read and rubocop wont complain about it